tour_model_eval/viz_similarity_unlabeled.ipynb

```diff
@@ -73,7 +81,7 @@
    "outputs": [],
    "source": [
     "# we should really plot this for all the uuids, but let's start with one at a time\n",
-    "first_uuid = long_term_uuid_list[0]"
+    "first_uuid = all_users[0]"
```

Comment:
we can remove the lines with the `first_uuid` since they were only examples, right?

Reply:
If a remove the the lines with the `first_uuid`, should I also remove 
```sim = similarity.similarity(data, radius)```
```sim.bin_data()```
```sim.calc_cutoff_bins()```
```graph(sim)```
above the for loop?